src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8252,6 +8288,163 @@ namespace UnitarySystems {
                 if ((HeatingLoad && ZoneLoad > SensOutputOff) || (CoolingLoad && ZoneLoad < SensOutputOff)) {
                     Real64 SensOutput;
                     Real64 LatOutput;
+                    if (this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_SubcoolReheat) {
+                        if (CoolingLoad && this->LoadSHR > 0.0) {
+                            int CoilInletNode = coilCoolingDXs[this->m_CoolingCoilIndex].evapInletNodeIndex;
+                            this->CoilSHR = 0.0;
+                            Real64 totalRate = 0.0;
+                            Real64 minAirHumRat;
+                            Real64 sensRate = 0.0;
+                            Real64 latRate = 0.0;
+                            // Real64 latLoad;
+                            Real64 LowSpeedEnthalpy;
+                            Real64 LowSpeedTemp;
+                            Real64 LowSpeedHumRat;
```

Comment:
Minor: These 3 variables aren't used,  just set equal to the node condition and then nothing done with this information.

Reply:
@rraustad A good catch. There variables are removed.